frontend/src/shared/helpers.tsx

```diff
@@ -804,9 +815,25 @@ export const CohortSampleDetailsColumns: ColDef[] = [
   },
 ];
 
-setupEditableSampleFields(SampleDetailsColumns);
+export const ReadOnlyCohortSampleDetailsColumns = _.cloneDeep(
+  CohortSampleDetailsColumns
+);
+
+setupEditableSampleFields(SampleMetadataDetailsColumns);
 setupEditableSampleFields(CohortSampleDetailsColumns);
 
+const seenColumns = new Set();
+export const combinedSampleDetailsColumns = [
+  ...SampleMetadataDetailsColumns,
+  ...ReadOnlyCohortSampleDetailsColumns,
+].filter((col) => {
+  if (seenColumns.has(col.field)) {
+    return false;
+  }
+  seenColumns.add(col.field);
+  return true;
+});
+
```

Comment:
Created a read-only copy of `CohortSampleDetailsColumns`, then combined that with `SampleMetadataDetailsColumns` to form a variable with *unique* column definitions.